scheduler/src/cook/mesos/api.clj

```diff
@@ -1350,6 +1350,7 @@
 (def UpdateRetriesRequest
   {(s/optional-key :job) s/Uuid
    (s/optional-key :jobs) [s/Uuid]
+   (s/optional-key :groups) [s/Uuid]
```

Comment:
I thought we decided to add a flag for only retrying non-completed jobs in the group

Reply:
I'm going to do that as a follow-on. Thanks for the reminder though!

Reply:
Cool, do we have an issue for that?

Reply:
It's in JIRA.